library/src/libdaemonjvm/internal/LockProcess.scala

```diff
@@ -8,13 +8,12 @@ trait LockProcess {
 object LockProcess {
   class Default extends LockProcess {
```

Comment:
nNote that this change likely means we could inline/remove the whole `LockProcess` trait, as well as change `Int` to `Long`. However, I choose to maintain bincompat instead and just do the minimum here